arm/ui/routes.py

```diff
@@ -52,7 +52,11 @@ def rips():
 
 @app.route('/history')
 def history():
-    jobs = Job.query.filter_by()
+    if os.path.isfile(cfg['DBFILE']):
```

Comment:
If you run the web UI before any jobs are queued, it corrupts your DB and takes the whole system down.  I added this code, which is also in the `index` route, to prevent this.